src/src.js

```diff
@@ -140,7 +140,7 @@ const SVG = document.getElementById("S"),
               },
             },
             {
-              text: () => "Home",
+              text: () => _("Home"),
```

Comment:
this is not necessary, the string gets translated inside say() function, just adding the translation was already enough now!
```suggestion
              text: () => "Home",
```